src/main/kotlin/ar/edu/unq/ttip/llegarafindemes/jobs/PlazoFijoJob.kt

```diff
@@ -0,0 +1,30 @@
+package ar.edu.unq.ttip.llegarafindemes.jobs
+
+import ar.edu.unq.ttip.llegarafindemes.services.BCRAService
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.scheduling.annotation.EnableScheduling
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component
+@EnableScheduling
+class PlazoFijoJob {
+    @Autowired
+    private lateinit var bcraService: BCRAService
+
+    //TODO: Modificar periodicidad para que corra una vez al dia tras ser ejecutada la aplicacion
+    @Scheduled(cron = "0 0/2 * * * ?")
```

Comment:
Ojo que quedó cada 2 minutos (si no me equivoco). Podría ser
```suggestion
    @Scheduled(cron = "0 0 0 * * ?")
```
Si no le pifio, eso debería ser todos los días a la media noche.